pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -2076,7 +2081,7 @@ def get_power_bandwidth(self, trace, percent):
         """,
         validator=joined_validators(strict_discrete_set, truncated_discrete_set),
         values=[["AUTO", "MAN"], arange(10, 2e6)],
-        set_process=lambda v: v if isinstance(v, str) else int(v),
+        set_process=lambda v: v if isinstance(v, str) else str(int(v)) + " Hz",
```

Comment:
You could use an f-string as an alternative:
```suggestion
        set_process=lambda v: v if isinstance(v, str) else f"{int(v)} Hz",
```